src/dstack/_internal/core/models/common.py

```diff
@@ -45,3 +45,12 @@ def parse(cls, v: Union[int, str]) -> "Duration":
             }[unit]
             return cls(amount * multiplier)
         raise ValueError(f"Cannot parse the duration {v}")
+
+
+def is_core_model_instance(instance: CoreModel, class_: Type[CoreModel]) -> bool:
+    """
```

Comment:
I propose renaming the variables `instance` and `class_` to correspond with [original `isinstance`](https://docs.python.org/3/library/functions.html#isinstance) (`object` and `classinfo`).

Reply:
Since it's a positional-only C func, doesn't make a difference. `object` can't be used.

BTW, the pylance signature:
```
def isinstance(__obj: object, __class_or_tuple: _ClassInfo) -> bool: ...
```

Reply:
Thank you!